src/player.js

```diff
@@ -147,6 +148,11 @@ class Player {
             screenfull.on('fullscreenchange', this.fullscreenchangeHandler);
         }
 
+        // TODO: NEED TO DETERMINE
+        // 1  IF addClipMarkup CAN BE CALLED FROM HERE AND
+        // 2  WHETHER WE SHOULD ONLY CALL addClipMarkup CONDITIONALLY
```

Comment:
I definitely think this should be an opt-in in some way. Especially for people who are referencing a version that is automatically updated to the latest. It's not nice to modify the page without explicit permission.

Reply:
@bdougherty @jdreetz we're only going to inject this markup for public videos, so that's one sort of layer of protection. Are y'all thinking along the lines of an additional option the the embed settings?

<img width="571" alt="image" src="https://user-images.githubusercontent.com/100798704/163455048-c7ee51f1-25c0-4340-8e50-b9a442ddf2e9.png">

Reply:
Yes, something like that. Enable/Disable depending on whether this is an opt-in or opt-out feature.

Reply:
Got it, I'll talk to Alex about it when he gets back. Implementation is _probably_ pretty straightforward.

Reply:
Got it, I'll talk to Alex about it when he gets back. Implementation is _probably_ pretty straightforward.

Reply:
@jdreetz @bdougherty following up: I spoke to Alex about it, and they don't intent to make it an opt-in.

Reply:
So is there going to be an opt-out?

Reply:
Hey Brad. According to Alex, we're not going to make it optional.